package.json

```diff
@@ -17,19 +17,39 @@
   ],
   "private": true,
   "dependencies": {
+    "@babel/core": "7.4.3",
+    "@babel/preset-env": "7.4.3",
+    "@babel/register": "7.4.4",
+    "assert": "^2.0.0",
+    "base64url": "^3.0.1",
+    "bigi": "^1.4.2",
+    "bs58": "^4.0.1",
+    "cbor": "3.0.3",
+    "create-hash": "^1.2.0",
+    "create-hmac": "^1.1.7",
+    "ecurve": "^1.0.6",
+    "elliptic": "^6.5.1",
+    "eosjs": "21.0.1-rc1",
+    "express": "4.17.1",
     "focus-visible": "4.1.5",
     "prop-types": "15.7.2",
+    "randombytes": "^2.1.0",
     "react": "16.8.6",
     "react-dom": "16.8.6",
+    "react-scripts": "3.2.0",
     "ual-eosio-reference-authenticator": "0.1.3",
     "ual-lynx": "0.2.2",
-    "ual-reactjs-renderer": "0.1.6",
+    "ual-reactjs-renderer": "0.1.7",
     "ual-scatter": "0.1.6",
```

Comment:
`ual-scatter` version needs to be updated with https://github.com/EOSIO/ual-scatter/pull/59 due to breaking changes. All other dependencies have been tested with eosjs `21.0.1-rc1`.